api/src/main/java/org/openmrs/util/VelocityExceptionHandler.java

```diff
@@ -43,4 +44,10 @@ public Object methodException(Class claz, String method, Exception e) throws Exc
 		throw e;
 	}
 	
+	@Override
+	public Object methodException(Context context, Class claz, String method, Exception e, Info info) {
```

Comment:
Is this change related to updating apache velocity?

Reply:
yes @dkayiwa  because the updating the apache velocity made changes in the code and brought about some errors during compilation that needed to be fixed